ci-operator/step-registry/telcov10n/functional/cnf-network/ocp-deploy/telcov10n-functional-cnf-network-ocp-deploy-ref.yaml

```diff
@@ -0,0 +1,54 @@
+ref:
+  as: telcov10n-functional-cnf-network-ocp-deploy
+  from_image:
+    namespace: telcov10n-ci
+    name: eco-ci-cd
+    tag: eco-ci-cd
+  commands: telcov10n-functional-cnf-network-ocp-deploy-commands.sh
+  timeout: 4h
+  resources:
+    requests:
+      cpu: 100m
+  credentials:
+  - namespace: test-credentials
+    name: telcov10n-ansible-group-all
+    mount_path: /var/group_variables/all
+  - namespace: test-credentials
+    name: telcov10n-ansible-group-bastions
+    mount_path: /var/group_variables/bastions
+  - namespace: test-credentials
+    name: telcov10n-ansible-group-hypervisors
+    mount_path: /var/group_variables/hypervisors
+  - namespace: test-credentials
+    name: telcov10n-ansible-group-kni-qe-92-masters
```

Comment:
Maybe we can reduce the number of secrets? Do we need `masters` and `master0-2`? same for nodes

Reply:
masters are common variables for for all masters and master0-2 unique for each control plane node